### PR TITLE
Skill - Glacies

### PR DESCRIPTION
**Pull request by JesterWizard**
Add unit's resistance to damage dealt (Skill % activation).

### CODE REVIEW

File: Wizardry/Core/BattleSys/Source/BattleHit.c [hunk 1]
```diff
@@ -48,6 +48,14 @@ int CalcBattleRealDamage(struct BattleUnit * attacker, struct BattleUnit * defen
         damage += defender->battleDefense / 4;
 #endif
 
+#if defined(SID_Glacies) && (SID_Glacies < MAX_SKILL_NUM)
+    if (CheckBattleSkillActivte(attacker, defender, SID_Glacies, 100))
```
Comment (MokhaLeee) [inline Wizardry/Core/BattleSys/Source/BattleHit.c:52]: 100?

**Comment by MokhaLeee** [comment]
also better not to put it to real-damage

File: Wizardry/Core/BattleSys/Source/BattleHit.c (hunk 1 above)
Comment (JesterWizard) [reply Wizardry/Core/BattleSys/Source/BattleHit.c]: Fixed

**Comment by JesterWizard** [comment]
> also better not to put it to real-damage

Changed its location

**Pull request merged**